src/code/z_play.c

```diff
@@ -483,7 +483,8 @@ void Play_Init(GameState* thisx) {
     Camera_InitDataUsingPlayer(&this->mainCamera, player);
     Camera_RequestMode(&this->mainCamera, CAM_MODE_NORMAL);
 
-    playerStartBgCamIndex = PARAMS_GET_U(player->actor.params, 0, 8);
+    playerStartBgCamIndex = PLAYER_GET_START_BG_CAM_INDEX(&player->actor);
+
     if (playerStartBgCamIndex != 0xFF) {
```

Comment:
Yeah I tried using -1 in the comparison but it doesnt match. If there is a way to match it then I would gladly change it, but I dont know if there is

Reply:
Did you try casting to u8?
`if (playerStartBgCamIndex != (u8)-1) {`

Reply:
It does match.
Maybe `#define PLAYER_START_BG_CAM_DEFAULT (u8)-1` should exist. I hate how long and clunky it is, but I think this is inherently a problem with the name `bgCamIndex` (which I want to explore renaming)

Reply:
yeah, technically withour the cast `-1 == 0xFFFFFFFF` because of `s32 playerStartBgCamIndex`

Reply:
Unless you plan on dropping BG from the name, I don't see how a rename wouldn't just make the name longer.

Reply:
well, the original name for these is "cam id", which is alot shorter than "bg cam index".
I think in general it shouldnt be called an "index", the fact that it indexes an array is not helpful.

Not that "cam id" is the best name, but something like it seems appropriate.
But anyway, this conversation should happen separately.

Reply:
Anyway, I added `PLAYER_START_BG_CAM_DEFAULT`. The word "index" is omitted for brevity and as a personal protest against it being in the name :)

Reply:
it would be more consistent if either `PLAYER_GET_START_BG_CAM_INDEX` uses `PARAMS_GET_S` and `PLAYER_START_BG_CAM_DEFAULT` is `(s8)-1`, or `PLAYER_START_BG_CAM_DEFAULT` is 0xFF (which is maybe not bad since it's hidden in a macro)

(also macro definition should be in parens like `((u8)-1)` here)